### ISSUE TITLE
QVGE SNAP on Manjaro

### ISSUE THREAD

**Issue by cgraham149** [ArsMasiuk/qvge#160]
QVGE crashes on start and displays these messages...

```
WARNING: cgroup v2 is not fully supported yet, proceeding with partial confinement

(qvgeapp:5508): dbind-WARNING **: 19:25:35.695: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-KC1WPPWSyF: No such file or directory
Gtk-Message: 19:25:35.764: Failed to load module "canberra-gtk-module"
Gtk-Message: 19:25:35.772: Failed to load module "canberra-gtk-module"
Qt: Session management error: None of the authentication protocols specified are supported
QSettings::Status(NoError)
Fontconfig warning: FcPattern object weight does not accept value [0 205)
zsh: segmentation fault (core dumped)
```

Strangely, it was working when I installed it but something may have been upgraded since and now this happens. Any ideas?

**Comment by ArsMasiuk** [comment]
Hi,
This is quite strange since qvge does not use dbus directly in any ways. Therefore
it looks for me as a system issue rather.
Btw which Version of qvge do you use? Have you tried to build 0.7 from
sources?
Best regards
Ars

**Comment by alexmyczko** [comment]
/me working on an official debian package, if that helps? http://sid.ethz.ch/debian/qvge/ meanwhile it entered debian officially…
